### PR TITLE
Update to 9.3.2

### PR DESCRIPTION
**Pull request by tatianacv**
Fixes https://github.com/ponder-lab/Pydev/issues/7

### CODE REVIEW

**Comment by tatianacv** [comment]
Note: The GH workflow has to be changed to pydev_9_3 instead of 9_3_1 for it to run the check. But since I didn't know if it pertained to this PR. Could you please let me know if I should add this @khatchad? For reference, this is where we would have to make the change. https://github.com/tatianacv/Pydev/blob/pydev_9_3/.github/workflows/tests-pydev.yml#L7-L11

**Comment by tatianacv** [comment]
> Note: The GH workflow has to be changed to pydev_9_3 instead of 9_3_1 for it to run the check. But since I didn't know if it pertained to this PR. Could you please let me know if I should add this @khatchad? For reference, this is where we would have to make the change. https://github.com/tatianacv/Pydev/blob/pydev_9_3/.github/workflows/tests-pydev.yml#L7-L11

I added this change in [171894f](https://github.com/ponder-lab/Pydev/pull/9/commits/171894fc2d6c3921f03c6354e8f21b890e9c8e45)

File: .github/workflows/tests-pydev.yml [hunk 1]
```diff
@@ -4,11 +4,11 @@ on:
   push:
     branches:
       - master
-      - pydev_9_3_1
+      - pydev_9_3
```
Comment (khatchad) [inline .github/workflows/tests-pydev.yml]: Agreed that this should really not be part of this PR. I'll change it separately.

Comment (tatianacv) [reply .github/workflows/tests-pydev.yml]: Okay, I will revert that commit.

**Pull request merged**